MH4U Dex/MonsterDetailsViewController.m

```diff
@@ -0,0 +1,72 @@
+//
+//  MonsterDetailsViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/9/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "MonsterDetailsViewController.h"
+#import "MonsterOverviewViewController.h"
+#import "MonsterLowRankTableViewController.h"
+
+typedef NS_ENUM(NSInteger, MonsterSegmentedControlPage) {
+    Overview = 0,
+    LowRank = 1,
```

Comment:
These names might be a bit too generic—they don't really indicate that the constants refer to the monster segmented control indicies.